builtin/game/chatcommands.lua

```diff
@@ -468,7 +468,7 @@ local function emergeblocks_progress_update(ctx)
 end
 
 core.register_chatcommand("emergeblocks", {
-	params = "(here [radius]) | (<pos1> <pos2>)",
+	params = "(here [<radius>]) | ((<pos1>) (<pos2>))",
```

Comment:
This and the 2 commands below expect pos1 pos2 in the form
(x1, y1, z1) (x2, y2, z2)
I feel the double-bracketing here confuses showing the scope of OR with having to enter "(X,Y,Z)".
Better i ithink to remove the inner brackets and make the format clear in the description.